packages/foreman/foreman/foreman.spec

```
@@ -772,13 +776,17 @@ install -Dp -m0644 extras/systemd/%{dynflow_sidekiq_service_name}.service %{buil
 install -Dp -m0755 script/%{name}-debug %{buildroot}%{_sbindir}/%{name}-debug
 install -Dp -m0755 script/%{name}-rake %{buildroot}%{_sbindir}/%{name}-rake
 install -Dp -m0755 script/%{name}-tail %{buildroot}%{_sbindir}/%{name}-tail
+install -Dp -m0755 script/%{name}-puma-status %{buildroot}%{_sbindir}/%{name}-puma-status
 install -Dp -m0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/logrotate.d/%{name}
 install -Dp -m0644 %{SOURCE4} %{buildroot}%{_sysconfdir}/cron.d/%{name}
 install -Dp -m0644 %{SOURCE5} %{buildroot}%{_tmpfilesdir}/%{name}.conf
 install -Dp -m0644 extras/systemd/%{name}.service %{buildroot}%{_unitdir}/%{name}.service
 install -Dp -m0644 extras/systemd/%{name}.socket %{buildroot}%{_unitdir}/%{name}.socket
 
+%if %{?scl:1}%{!?scl:0}
 sed -i '/^ExecStart/ s|/usr/bin/sidekiq \(.\+\)$|/usr/bin/scl enable tfm "sidekiq \1"|' %{buildroot}%{_unitdir}/%{dynflow_sidekiq_service_name}.service
+sed -i '/^puma-status/ s|puma-status \(.\+\)$|/usr/bin/scl enable tfm "puma-status \1"|' %{buildroot}%{_sbindir}/%{name}-puma-status
```

Wouldn't this also work?

```suggestion
sed -i '/^puma-status/ s|\(puma-status .\+\)$|/usr/bin/scl enable tfm "\1"|' %{buildroot}%{_sbindir}/%{name}-puma-status
```

---

Don't know, I can test it if this version prevents some bug or makes a big difference in the way it works?